pkg/config/secret.go

```diff
@@ -19,6 +19,7 @@ package config
 // Secret special type for storing secrets.
 type Secret string
 
+// this is test
```

Comment:
I think we should remove this before merge.

Reply:
yes, this is here just for testing. Once PR is approved, will remove it before merging.